lib/mayaUsd/fileio/orphanedNodesManager.cpp

```diff
@@ -141,8 +144,10 @@ void renamePullInformation(
         }
     }
 
-    const MDagPath& mayaPath = trieNode->data().editedAsMayaRoot;
-    TF_VERIFY(writePullInformation(pulledPath, mayaPath));
+    for (const PullVariantInfo& info : trieNode->data()) {
+        const MDagPath& mayaPath = info.editedAsMayaRoot;
+        TF_VERIFY(writePullInformation(pulledPath, mayaPath));
+    }
```

Comment:
There are a lot of changes like this in the orphan manager where what use to be a single value is now a vector because we support a single prim path being under different variants.